apps/netconf/netconf_main.c

```diff
@@ -449,12 +449,16 @@ netconf_input_frame(clicon_handle h,
     return retval;
 }
 
-/*! Look for an endtag in a cubf
- *  @param[in]     cb           The cbuf holding the received message
- *  @retval        0            No end tag has been detected
- *  @retval        1            End tag has been detected
+/*! @brief          Detect a given character sequence at the end of a \b cbuf
```

Comment:
brief/detail doxygen not used anywhere else in clixon.
doxygen seems to figure that out itself giving the first line as brief and after a \n uses the remaining text as detail?